README.md

```diff
@@ -1,2 +0,0 @@
-# automation
-for project: Project_myweb
```

Comment:
Add some README content.
README should contains at least info about what is the content/purpose of this repository